packages/vulcan-backoffice/package.js

```diff
@@ -0,0 +1,18 @@
+Package.describe({
+  name: 'vulcan:backoffice',
+  summary: 'Vulcan automated backoffice generator',
+  version: '1.13.3',
```

Comment:
version is wrong now it should be 1.14.1 I think (same  as core after a merge from devel)